### PR TITLE
feat(APIInviteGuild): add boost count

### PR DESCRIPTION
**Pull request by suneettipirneni**
**Please describe the changes this PR makes and why it should be merged:**

This also fixes the guild on invites to be accurate to the example in the docs.

**Reference Discord API Docs PRs or commits:**
- https://github.com/discord/discord-api-docs/pull/4493

### CODE REVIEW

File: payloads/v9/invite.ts [hunk 1]
```diff
@@ -3,12 +3,33 @@
...
 import type { APIUser } from './user';
 import type { APIGuildScheduledEvent } from './guildScheduledEvent';
 
+export interface APIInviteGuild
+	extends Pick<
+		APIGuild,
+		| 'id'
+		| 'name'
+		| 'splash'
+		| 'banner'
```
Comment (vladfrangu) [inline payloads/v9/invite.ts:18]: If these fields (and icon and vanity url are always null, don't pick them from the base guild interface

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: deno/payloads/v8/invite.ts [hunk 2]
```diff
@@ -3,12 +3,23 @@
...
 import type { APIInviteStageInstance } from './stageInstance.ts';
 import type { APIUser } from './user.ts';
 import type { APIGuildScheduledEvent } from './guildScheduledEvent.ts';
 
+export interface APIInviteGuild
+	extends Pick<
+		APIGuild,
+		'id' | 'name' | 'description' | 'features' | 'verification_level' | 'nsfw_level' | 'premium_subscription_count'
+	> {
+	splash: null;
```
Comment (vladfrangu) [inline deno/payloads/v8/invite.ts:17]: Are you sure these are null? Can you fetch, say, djs's invite and see what it returns? 👀

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: deno/payloads/v8/invite.ts (hunk 2 above)
Comment (GoldenAngel2) [reply deno/payloads/v8/invite.ts:17]: `splash`, `banner`, `icon` and `vanity_url_code` gets returned

Comment (suneettipirneni) [reply deno/payloads/v8/invite.ts:17]: thanks, I'll add those back

**Review by zmkq — APPROVED** [review]

**Review by vladfrangu — APPROVED** [review]

**Comment by suneettipirneni** [comment]
Upstream PR merged

**Pull request merged**